.github/workflows/ci.yml

```diff
@@ -60,6 +60,8 @@ jobs:
           ghc: 'latest'
         - os: macOS-latest
           ghc: 'latest'
+        - os: ubuntu-16.04
```

Comment:
Maybe include the motivation for using this specific platform as a comment?!

Reply:
Done, thanks.